web/moduly/finance.php

```diff
@@ -89,11 +81,8 @@
     <?php } ?>
     <ul class="seznam-bez-okraje">
     <li class="poznamka">Při pozdější platbě tě systém dne <strong>1.7</strong> (příp. 16.7. při pozdější přihlášce)<strong> automaticky odhlásí</strong>.</li>
-    <li class="poznamka">Pokud si aktivity (znovu) přihlásíš po 30.6., beze slevy nebo na místě pak platíš <?=$pozde?>.</li>
     <li class="poznamka">Při plánování aktivit si na účet pošli klidně více peněz. Přebytek ti vrátíme na infopultu nebo ho můžeš využít k přihlašování uvolněných aktivit na místě.</li>
     </ul>
-  <?php }else{ ?>
```

Comment:
Vyhozením else větve se tam vytvořil stav, že ten text někdy úplně zmizí. Vzhledem k tomu, že je tam info o druhé vlně odhlašování, tak to asi není úplně v pohodě.